osu.Game.Rulesets.Osu/Objects/Drawables/DrawableOsuHitObject.cs

```diff
@@ -51,6 +52,19 @@ protected virtual void UpdatePreemptState()
         protected virtual void UpdateCurrentState(ArmedState state)
         {
         }
+
+        private readonly WeakReference<OsuInputManager> osuActionInputManager = new WeakReference<OsuInputManager>(null);

```

Comment:
hmm, i'm not sure the weak reference would ever be used. was there a specific reason for it?